frontend/apps/marketing/src/components/header/Header.tsx

```diff
@@ -14,212 +15,18 @@ import musicLabImage from '@public/images/header-music-lab-icon.png';
-      image: appLabImage.src,
-      description: 'Make apps with Javascript',
-    },
-    {
-      key: 'gameLab',
-      label: 'Game Lab',
-      href: 'https://studio.code.org/projects/gamelab/new',
-      image: gameLabImage.src,
-      description: 'Build simple games',
-    },
-    {
-      key: 'musicLab',
-      label: 'Music Lab',
-      href: '/music',
-      image: musicLabImage.src,
-      description: 'Create music with code',
-    },
-    {
-      key: 'danceParty',
-      label: 'Dance Party',
-      href: 'https://studio.code.org/projects/dance/new',
-      image: dancePartyImage.src,
-      description: 'Make a dance party with AI',
-    },
-    {
-      key: 'pythonLab',
-      label: 'Python Lab',
-      href: 'https://studio.code.org/projects/pythonlab/new',
-      image: pythonLabImage.src,
-      description: 'Code using Python',
-    },
-    {
-      key: 'viewAllProjects',
-      label: 'View All Projects',
-      href: 'https://studio.code.org/projects',
-      image: allProjectsImage.src,
-    },
-  ],
-  accountLinks: {
-    signIn: {
-      label: 'Sign In',
-      href: 'https://studio.code.org/users/sign_in',
-    },
-    createAccount: {
-      label: 'Create Account',
-      href: 'https://studio.code.org/users/sign_up/account_type',
-    },
-    goToDashboard: {
-      label: 'Go to Dashboard',
-      href: 'https://studio.code.org/home',
-    },
-  },
-  isLoggedIn: false,
-  helpButtonLabel: {
-    open: 'Open Help menu',
-    close: 'Close Help menu',
-    menu: 'Help menu',
-  },
-  helpLinks: [
-    {
-      key: 'helpAndSupport',
-      label: 'Help and support',
-      href: 'https://support.code.org/',
-    },
-    {
-      key: 'reportAProblem',
-      label: 'Report a problem',
-      href: 'https://support.code.org/hc/en-us/requests/new',
-    },
-  ],
-  hamburgerButtonLabel: {
-    open: 'Open Hamburger menu',
-    close: 'Close Hamburger menu',
-    menu: 'Hamburger menu',
-  },
-  hamburgerLinks: [
-    {
-      key: 'learn',
-      label: 'Learn',
-      href: '/students',
-      hasDisplayLogic: true,
-    },
-    {
-      key: 'teach',
-      label: 'Teach',
-      href: '/teach',
-      hasDisplayLogic: true,
-    },
-    {
-      key: 'districts',
-      label: 'Districts',
-      href: '/administrators',
-      hasDisplayLogic: true,
-    },
-    {
-      key: 'stats',
-      label: 'Stats',
-      href: '/promote',
-    },
-    {
-      key: 'donate',
-      label: 'Donate',
-      href: '/help',
-    },
-    {
-      key: 'incubator',
-      label: 'Incubator',
-      href: '/incubator',
-    },
-    {
-      key: 'about',
-      label: 'About',
-      href: '/about',
-    },
-    {
-      key: 'helpAndSupport',
-      label: 'Help and support',
-      href: 'https://support.code.org/',
-    },
-    {
-      key: 'reportAProblem',
-      label: 'Report a problem',
-      href: 'https://support.code.org/hc/en-us/requests/new',
-    },
-  ],
-};
+const defaultProps = getDefaultProps({
```

Comment:
nit: Would be good to rename this as `getDefaultHeaderProps` to be more explicit? I can see how getDefaultProps gets confusing